tappsbt/decode.go

```diff
@@ -230,39 +230,60 @@ func (o *VOutput) decode(pOut psbt.POutput, txOut *wire.TxOut) error {
 	}
 
 	anchorOutputIndex := uint64(o.AnchorOutputIndex)
-	mapping := []decoderMapping{{
-		key:     PsbtKeyTypeOutputTapType,
-		decoder: tlvDecoder(&o.Type, vOutputTypeDecoder),
-	}, {
-		key:     PsbtKeyTypeOutputTapIsInteractive,
-		decoder: booleanDecoder(&o.Interactive),
-	}, {
-		key:     PsbtKeyTypeOutputTapAnchorOutputIndex,
-		decoder: tlvDecoder(&anchorOutputIndex, tlv.DUint64),
-	}, {
-		key:     PsbtKeyTypeOutputTapAnchorOutputInternalKey,
-		decoder: tlvDecoder(&o.AnchorOutputInternalKey, tlv.DPubKey),
-	}, {
-		key:     PsbtKeyTypeOutputTapAnchorOutputBip32Derivation,
-		decoder: bip32DerivationDecoder(&o.AnchorOutputBip32Derivation),
-	}, {
-		key: PsbtKeyTypeOutputTapAnchorOutputTaprootBip32Derivation,
-		decoder: taprootBip32DerivationDecoder(
-			&o.AnchorOutputTaprootBip32Derivation,
-		),
-	}, {
-		key:     PsbtKeyTypeOutputTapAsset,
-		decoder: assetDecoder(&o.Asset),
-	}, {
-		key:     PsbtKeyTypeOutputTapSplitAsset,
-		decoder: assetDecoder(&o.SplitAsset),
-	}, {
-		key: PsbtKeyTypeOutputTapAnchorTapscriptSibling,
-		decoder: tlvDecoder(
-			&o.AnchorOutputTapscriptSibling,
-			commitment.TapscriptPreimageDecoder,
-		),
-	}}
+	mapping := []decoderMapping{
+		{
```

Comment:
Is this to make editing easier in vim? Because tbh, I like the more compact form, as it also saves one level of indentation.

Reply:
Don't think it's a vim specific thing, imo this way I can just copy/paste a row to add another one, then edit that. Before you had to mind the double curly braces at the end, and also the way each row was sort of folded under itself.